botan/tests/tests.rs

```diff
@@ -770,6 +770,47 @@ fn test_pubkey_sign() -> Result<(), botan::Error> {
     Ok(())
 }
 
+#[test]
+fn test_pubkey_sign_der() -> Result<(), botan::Error> {
+    let msg = vec![1, 23, 42];
+
+    let mut rng = botan::RandomNumberGenerator::new_system()?;
+
+    let ecdsa_key = botan::Privkey::create("ECDSA", "secp256r1", &mut rng)?;
+    assert!(ecdsa_key.key_agreement_key().is_err());
+
+    let mut signer = botan::Signer::new_with_der_formatted_signatures(&ecdsa_key, "SHA-256")?;
```

Comment:
For Botan2 these must be "EMSA1(SHA-256)", which explains the `NotImplemented` in CI